src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/totrinnskontroll/domain/Totrinnskontroll.kt

```diff
@@ -28,4 +28,6 @@ data class Årsaker(
 enum class TotrinnsKontrollStatus {
     UNDERKJENT,
     KLAR,
+    ANGRE_SEND_TIL_BESLUTTER,
+    SKAL_TOTRINNSKONTROLLERES,
```

Comment:
Kanskje oppdatere disse til:
```
UNDERKJENT,
GODKJENT
ANGRET,
SKAL_VURDERES eller KAN_FATTE_VEDTAK
```

Reply:
Liten definisjon her.  Føler at når saksbehandler oppretter totrinnskontroll ved å sende den til beslutter, så synes eg at status bør vere SKAL_VURDERES, og når det blir satt på/tilordna beslutter så skifter den status til KAN_FATTE_VEDTAK. Eller der dette ein tankefeil frå mi side at det er to forskjellige tilstander?

Reply:
valgt KAN_FATTE_VEDTAK